library/formsoptions.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/*
+ * Functions to be used in the forms.
```

Comment:
Noooo.  Nothing that deep.  Just suggesting a bigger explain'n of what the file does.